src/modules/mc_hover_thrust_estimator/hover_thrust_estimator_params.c

```diff
@@ -129,4 +129,4 @@ PARAM_DEFINE_FLOAT(HTE_VZ_THR, 2.0);
  * @unit normalized_thrust
  * @group Hover Thrust Estimator
  */
-PARAM_DEFINE_FLOAT(HTE_THR_RANGE, 0.2);
+PARAM_DEFINE_FLOAT(HTE_THR_RANGE, 0.3);
```

Comment:
I think it would be better if this was configured in the airframe config, rather than changing the default value since this is shared across many different vehciles